docs/reference-implementation/Brownfield-Considerations.md

```diff
@@ -0,0 +1,29 @@
+# Brownfield scenario considerations
+
+While the Azure Mission-Critical online reference implementation is considered to be deployed mostly as part of Proof-of-Concept deployments into empty Azure subscription(s) does it still provide a toolbox of valuable artifacts for brownfield scenarios. This document contains the main considerations of how the online reference implementation can be deployed into existing environments.
```

Comment:
I would probably not state it like this regarding "this is mostly just for PoC". Sounds like it is not production-ready - which it is (mostly...)

Reply:
happy to rephrase/optimize it - just wanted to start with something

Reply:
Outdated.